.github/workflows/publish-release-artifacts.yml

```diff
@@ -17,6 +17,9 @@ jobs:
           echo "ShouldUpdateKrew:"${{ !github.event.release.prerelease && (steps.latest-release.outputs.latestTag == github.event.release.tag_name) }}
           echo "latestTag=$LATEST_VERSION" >> "$GITHUB_OUTPUT"
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
history is not needed?

Reply:
Wont we need all commits? Without this i think it pulls only latest commit?

Reply:
Why do we need all commits here? We're not interacting with the git history right?

Reply:
Ohh, Does it merge all commits into one commit. Just want to make sure all changes will be pulled.